### PR TITLE
[zh-cn]fix intro.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Follow up on https://github.com/kubernetes/website/issues/35059

Updated [Windows containers in Kubernetes](https://kubernetes.io/zh-cn/docs/concepts/windows/intro/) page.

### CODE REVIEW

File: content/zh-cn/docs/concepts/windows/intro.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ Kubernetes 关键组件在 Windows 上的工作方式与在 Linux 上相同。
 
 * {{< glossary_tooltip text="Services" term_id="service" >}}
 
-  有关更多详细信息，请参考[负载均衡和 Service](#load-balancing-and-services)。
+  有关更多详细信息，请参考[负载均衡和 Service](/zh-cn/docs/concepts/services-networking/windows-networking/#load-balancing-and-services)。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/windows/intro.md:235]: The commented original text should be updated too.

**Comment by Sea-n** [comment]
Hold until upstream change merged.
/hold

**Comment by yanrongshi** [comment]
> Hold until upstream change merged. /hold

The upstream has change

**Comment by yanrongshi** [comment]
@Sea-n cc

**Comment by Sea-n** [comment]
Please run `./scripts/lsync.sh content/zh-cn/docs/concepts/windows/intro.md` script.
There is another typo fix in upstream.

**Comment by yanrongshi** [comment]
@Sea-n cc

**Review by Sea-n — COMMENTED** [review]
Thanks, it's ready for merge! 🥳
/lgtm

**Pull request merged**